test/integration/controllers/DeviceCommandAPI.spec.js

```diff
@@ -0,0 +1,185 @@
+            .then(dataAll => {
+                for (const data of dataAll) {
+                    for (const deviceCommandKey in data) {
+                        testDeviceCommands[deviceCommandKey].id = data[deviceCommandKey].id;
+                        testDeviceCommands[deviceCommandKey].timestamp = data[deviceCommandKey].timestamp;
+                        testDeviceCommands[deviceCommandKey].lastUpdated = data[deviceCommandKey].lastUpdated;
+                        assert.containsAllKeys(data[deviceCommandKey], Object.keys(testDeviceCommands[0]));
+                    }
+                }
+            })
+            .then(done)
+            .catch(done);
+    });
+
+
+    it('DeviceCommand API.get()', done => {
+
+
+        Promise.all([httpDeviceHive.command.get(deviceId, testDeviceCommands[0].id), wsDeviceHive.command.get(deviceId, testDeviceCommands[0].id)])
+            .then(dataAll => {
+                const expected = testDeviceCommands[0];
+                for (const data of dataAll) {
+                    assert.isObject(data);
+                    assert.deepInclude(data, expected);
+                }
+            })
+            .then(done)
+            .catch(done);
+    });
+
+
+    it('DeviceCommandAPI.update()', done => {
+
+        const command = new DeviceHive.models.Command(testDeviceCommands[0], testDeviceCommands[0]);
+        const command2 = new DeviceHive.models.Command(testDeviceCommands[0], testDeviceCommands[1]);
+
+        Promise.all([httpDeviceHive.command.update(command), wsDeviceHive.command.update(command2)])
+            .then(() => done())
+            .catch(done);
+    });
+
+
+    it('DeviceCommandAPI.poll()', done => {
+
+        // Configurating Command List query
+        const commandPollQuery = new DeviceHive.models.query.CommandPollQuery({
+            deviceId,
+            returnUpdatedCommands: true,
+            limit: 1,
+            waitTimeout: 1
+        });
+
+        httpDeviceHive.command.poll(commandPollQuery)
+            .then(() => done())
+            .catch(done);
+
+        // emit command
+        setTimeout(() => {
+            const command = new DeviceHive.models.Command(testDeviceCommands[0]);
+            httpDeviceHive.command.insert(deviceId, command);
+        }, 50);
+    });
+
+
+    it('DeviceCommandAPI.pollMany()', done => {
+
+        const commandPollManyQuery = new DeviceHive.models.query.CommandPollManyQuery({
+            deviceIds: deviceId
+        });
+
+        httpDeviceHive.command.pollMany(commandPollManyQuery)
+            .then(() => done())
+            .catch(done);
+
+        // emit command
+        setTimeout(() => {
+            const command = new DeviceHive.models.Command(testDeviceCommands[0]);
+            httpDeviceHive.command.insert(deviceId, command);
+        }, 50);
+    });
+
+
+    it('DeviceCommandAPI.wait()', done => {
```

Comment:
You can omit second argument of it() and leave only test name so mocha will mark it as TODO for you